### ISSUE TITLE
DataFrame.plot(subplots=True) forces black-and-white mode regardless of other settings

### ISSUE THREAD

**Issue by Pastafarianist** [pydata/pandas#11536]
```python
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd

df = pd.DataFrame(np.random.randn(6, 4), index=pd.date_range('20150101', periods=6), columns=list('ABCD'))

plotters = [
	"df.plot()",
	"df.plot(legend=None)",
	"df.plot(legend=None, subplots=True, layout=(2, 2))",
	"df['A'].plot(legend=None)",
	"df['A'].plot(legend=None, subplots=True)"
]

for i, plotter in enumerate(plotters, 1):
	eval(plotter)
	plt.title(plotter)
	plt.savefig('%d.png' % i)
	plt.close('all')

```

Also, notice how the title is broken in the third image.

![1](https://cloud.githubusercontent.com/assets/1315874/11011823/43e5a7a2-84fd-11e5-91bf-0a82d0eca5ba.png)

![2](https://cloud.githubusercontent.com/assets/1315874/11011826/49813ffa-84fd-11e5-88a3-261f5c36c0ad.png)

![3](https://cloud.githubusercontent.com/assets/1315874/11011828/4e6623b4-84fd-11e5-9dfd-b1cd8e004056.png)

![4](https://cloud.githubusercontent.com/assets/1315874/11011832/58565524-84fd-11e5-9bf9-67d79d3f71f0.png)

![5](https://cloud.githubusercontent.com/assets/1315874/11011834/5d705ab4-84fd-11e5-92ad-87c53f6ce19c.png)

**Comment by sinhrks** [comment]
Thanks for the report. 

This should be fixed in 0.17.0 (#9894). Otherwise reopen attaching ``pd.show_versions``.

**Comment by Pastafarianist** [comment]
I have indeed been running this under Pandas 0.16.2, and I have forgotten to search for existing reports before submitting my own. Thank you for taking time to point me to the fact that the problem has already been fixed.

There is still, however, a problem in the third plot. I will create another issue for that one.
